src/d_main.c

```diff
@@ -1827,6 +1828,12 @@ void D_DoomMain(void)
   // [FG] emulate a specific version of Doom
   InitGameVersion();
 
+  char *extras = D_FindWADByName("extras.wad");
```

Comment:
Do we need to add any checks here? The name is too simple, there was also a version for the Unity port.

Reply:
Won't hurt to auto-load it even if it doesn't contain what we expect?

Reply:
> Won't hurt to auto-load it even if it doesn't contain what we expect?

Probably won't. But we need -noextras parameter just in case it annoys some users (for example, it contains DSSECRET sound or icons for weapon rolette).